features/async-clipboard.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Asynchronous Clipboard API
+category: api
+firefox_status: 63
+bugzilla: 1461465
+mdn_url: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
+spec_url: https://w3c.github.io/clipboard-apis/#async-clipboard-api
+spec_repo: https://github.com/w3c/clipboard-apis
+chrome_ref: 5861289330999296
+ie_ref: 'Clipboard API'
```

Comment:
wouldn't this break the report of what the IE status is, due to the fact that IE mixes the sync API (ClipboardEvent) and the async one in the same reference, but they shipped only the ClipboardEvent part ?

Reply:
good point; I've removed it for now.